modules/client/test/unit/client-common/utils.test.ts

```diff
@@ -44,6 +44,18 @@ const REMOVE_ADDRESSES_ACTION = {
     ),
   ),
 };
+const UPGRADE_TO_ACTION = {
```

Comment:
irrelevant here, since we are only checking the name of the function what we need are the first 8 chars of the encoding, the rest does not matter to us. Even if we decode the actions it wouldnt matter because we will only decode the `updateSettings`, `addAddresses` and `removeAdddresses`actions